From 516bbe6efaec4e510244509c80484910bc9408e5 Mon Sep 17 00:00:00 2001
From: Robin Gong <b38343@freescale.com>
Date: Tue, 3 Dec 2013 14:59:05 +0800
Subject: [PATCH 0571/5242] MLK-11703-6: ARM: imx6: Add charging led support
 on Sabresd board

commit  cfa6a44b5d2957ac90c5d41177d0f917a757fa00 from
https://source.codeaurora.org/external/imx/linux-imx.git

cherry-pick below patch:

ENGR00290496-1 ARM: imx6: Add charging led support on Sabresd board

Enable led lighting while the board in charging status. Implement it on Sabresd
board.

Signed-off-by: Robin Gong <b38343@freescale.com>
(cherry picked from commit 7a47183634c524f5ac11c60fe555b0b18c0fe7e0)
(cherry picked from commit 28b266aa0bdd531b8eb43a76435f2fcc658bab84)
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx6qdl-sabresd.dtsi |    9 +++++++++
 1 file changed, 9 insertions(+)

diff --git a/arch/arm/boot/dts/imx6qdl-sabresd.dtsi b/arch/arm/boot/dts/imx6qdl-sabresd.dtsi
index cc20c62..8ed9fe5 100644
--- a/arch/arm/boot/dts/imx6qdl-sabresd.dtsi
+++ b/arch/arm/boot/dts/imx6qdl-sabresd.dtsi
@@ -42,6 +42,15 @@
 		stdout-path = &uart1;
 	};
 
+	leds {
+		compatible = "gpio-leds";
+
+		charger-led {
+			gpios = <&gpio1 2 0>;
+			linux,default-trigger = "max8903-charger-charging";
+		};
+	};
+
 	memory@10000000 {
 		reg = <0x10000000 0x40000000>;
 	};
-- 
1.7.9.5

